c7n/filters/iamaccess.py

```diff
@@ -206,6 +206,12 @@ def handle_aws_sourcearn(self, s, op, key, values):
     def handle_aws_sourceip(self, s, op, key, values):
         return False
 
+    def handle_aws_sourcevpce(self, s, op, key, values):
```

Comment:
it looks like the 3 above implemented the same default logic, probably better to implement an default handler that has some list of default handled cases instead of writing the same thing 5 times.